config/jobs/check-prow-config/check-prow-config.yaml

```diff
@@ -1,22 +1,39 @@
 presubmits:
   falcosecurity/test-infra:
-    - name: check-prow-config
-      branches:
-        - ^master$
-      decorate: true
-      skip_report: false
-      always_run: true
-      spec:
-        containers:
-          - image: gcr.io/k8s-prow/checkconfig:v20220901-5db9cf5fa2
-            command:
-              - checkconfig
-            args:
-            - --config-path=config/config.yaml
-            - --job-config-path=config/jobs
-            - --plugin-config=config/plugins.yaml
-        nodeSelector:
-          Archtype: "x86"
+  - name: check-prow-config
+    branches:
+    - ^master$
+    decorate: true
+    skip_report: false
+    always_run: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-prow/checkconfig:v20220901-5db9cf5fa2
+        command:
+        - checkconfig
+        args:
+        - --config-path=config/config.yaml
+        - --job-config-path=config/jobs
+        - --plugin-config=config/plugins.yaml
+      nodeSelector:
+        Archtype: "x86"
+  falcosecurity/evolution:
+  - name: check-prow-config-evolution
```

Comment:
Should it run only when the `org.yaml` file has changed?

Reply:
I agree @leogr. Good catch! Thanks

Reply:
Applied